cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/ProgramOptionConstants.java

```diff
@@ -38,4 +38,10 @@
   public static final String SCHEDULE_NAME = "scheduleName";
```

Comment:
I guess I am just a little irritated that even for constants added in the same PR, we cannot be consistent (see the next couple of lines).